.gitignore

```
@@ -13,3 +13,4 @@
 /test-utilities/Cargo.lock
 /test-utilities/target
 /tmp
+.vscode/
```

This should be:

```suggestion
/.vscode
```

Also, the lines in this file should be sorted.